bin/dogescript.js

```diff
@@ -16,14 +16,34 @@ process.stdout.write("[dogescript@"+pjson.version+"]\n");
 
 if (argv._[0]) {
     var file = fs.readFile(path.resolve(process.cwd(), argv._[0]), {encoding: 'utf-8'}, function (err, script) {
-        if (argv['true-doge']) var lines = script.split(/ {3,}|\r?\n/);
-        else var lines = script.split(/\r?\n/);
+        if (argv['true-doge'])
+        {
+          process.stdout.write("!-- WARNING: true-doge mode is deprecated --!\n");
+          var lines = script.split(/ {3,}|\r?\n/);
+        }
+        else
+        {
+          var lines = script.split(/\r?\n/);
```

Comment:
Putting the declaration outside the if would look nicer, since in most languages it's required to be that way